source/extensions/filters/http/well_known_names.h

```diff
@@ -81,7 +81,7 @@ class HttpFilterNameValues {
   // AWS Lambda filter
   const std::string AwsLambda = "envoy.filters.http.aws_lambda";
   // OAuth filter
-  const std::string OAuth = "envoy.filters.http.oauth";
+  const std::string OAuth = "envoy.filters.http.oauth2";
```

Comment:
Yes. This is for consistency since we print the name of extensions at startup, i.e. `[info][main] [external/envoy/source/server/server.cc:309]: envoy.filters.http: ...`

- 

-